apps/klasses/models.py

```diff
@@ -17,14 +17,14 @@
 def upload_image(instance, filename):
     file_split = filename.split('.')
     file_extension = file_split[len(file_split) - 1]
-    path = "images/{0}/class_image.{1}".format(instance.course_number, file_extension)
+    path = "images/{0}/{1}/class_image.{2}".format(instance.pk, instance.course_number, file_extension)
```

Comment:
can we be using f strings on this project?

Reply:
Yes! I should have caught this!

Reply:
✅